api/src/pcapi/routes/backoffice/templates/chronicles/details.html

```diff
@@ -0,0 +1,194 @@
+                  {% if chronicle.isActive %}
+                    {{ build_modal_form("unpublish-chronicle-modal-" + chronicle.id|string, url_for('backoffice_web.chronicles.unpublish_chronicle', chronicle_id=chronicle.id) , chronicle_publication_form, "Dépublier", "Dépublier la chronique " + chronicle.id|string, "Dépublier") }}
+                  {% else %}
+                    {{ build_modal_form("publish-chronicle-modal-" + chronicle.id|string, url_for('backoffice_web.chronicles.publish_chronicle', chronicle_id=chronicle.id) , chronicle_publication_form, "Publier", "Publier la chronique " + chronicle.id|string, "Publier") }}
+                  {% endif %}
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="row pt-3">
+            <div class="col-4">
+              <div class="fs-6">
+                <p class="mb-1">
+                  <span class="fw-bold">Prénom :</span>
+                  {% if chronicle.firstName %}
+                    {{ chronicle.firstName }}
+                  {% else %}
+                    Non renseigné
+                  {% endif %}
+                  {% if chronicle.userId %}
+                    ( <a href={{ url_for("backoffice_web.public_accounts.get_public_account", user_id=chronicle.userId) }}>Voir le profil du jeune</a> )
+                  {% endif %}
+                </p>
+                <p class="mb-1">
+                  <span class="fw-bold">Age :</span>
+                  {% if chronicle.age %}
+                    {{ chronicle.age }} ans
+                  {% else %}
+                    Non renseigné
+                  {% endif %}
+                </p>
+                <p class="mb-1">
+                  <span class="fw-bold">Ville :</span>
+                  {% if chronicle.city %}
+                    {{ chronicle.city }}
+                  {% else %}
+                    Non renseigné
+                  {% endif %}
+                </p>
+                <p class="mb-1 text-nowrap">
+                  <span class="fw-bold">Email :</span> {{ chronicle.email }}
+                </p>
+              </div>
+            </div>
+            <div class="col-5">
+              <p class="mb-1">
+                <span class="fw-bold">Titre de l'œuvre :</span>
+                {% if product_name %}
+                  {{ product_name }}
+                {% else %}
+                  Non renseigné
+                {% endif %}
+              </p>
+              <p class="mb-1">
+                <span class="fw-bold">EAN :</span>
+                {% if chronicle.ean %}
+                  {{ chronicle.ean }}
+                {% else %}
+                  Non renseigné
+                {% endif %}
+              </p>
+            </div>
+            <div class="col-3">
+              <p class="mb-1">
+                <span class="fw-bold">Date de soumission :</span> {{ chronicle.dateCreated | format_date("Le %d/%m/%Y à %Hh%M") }}
```

Comment:
N'est-ce pas le formatage `format_date_time` que tu souhaites ?
```suggestion
                <span class="fw-bold">Date de soumission :</span> {{ chronicle.dateCreated | format_date_time }}
```